package.json

```
@@ -30,6 +30,7 @@
   "devDependencies": {
     "@types/common-tags": "1.8.1",
     "@types/diff": "5.0.3",
+    "@types/jest": "^29.5.0",
```

```suggestion
    "@types/jest": "29.5.0",
```

Will also need to run `npm i` after committing this to update the package-lock.json

---

Done.